src/richie/apps/core/context_processors.py

```diff
@@ -46,6 +46,17 @@ def site_metas(request):
     if getattr(settings, "CDN_DOMAIN", None):
         context["CDN_DOMAIN"] = settings.CDN_DOMAIN
 
+    # Add a MEDIA_URL_PREFIX to context to prefix the media url files to have an absolute URL
+    if settings.MEDIA_URL.startswith('//'):
+        # Eg. //my-cdn-user.cdn-provider.com/media/
+        context["MEDIA_URL_PREFIX"] = request.scheme + ":"
```

Comment:
Make it an f string as well?
```suggestion
        context["MEDIA_URL_PREFIX"] = f"{request.scheme:s}:"
```

Reply:
Ok